sgrep.yml

```diff
@@ -0,0 +1,26 @@
+rules:
```

Comment:
Does this have to be at top-level?  Ideally I'd want to put it in `tools/` to avoid cluttering the project's top-level directory.